printer-creality-cr10v3-bltouch.cfg

```diff
@@ -0,0 +1,229 @@
+#Stock Creality CR-10 V3 with BL-Touch and runout filament sensor
+[stepper_x]
+step_pin: PF0 #ar54
+dir_pin: PF1 #ar55
+enable_pin: !PD7 #!ar38
+microsteps: 16
+rotation_distance: 40 
+#Step distance is deprecated and will be removed in future releases
+#step_distance: .0125 <=> 0.0125 mm * 200 steps *16 microsteps = 40 mm
```

Comment:
Please remove this and similar comments.

Reply:
Comments were removed